services/search/search_engine.py

```diff
@@ -52,7 +52,7 @@ def query(self, query):
         lan = response['lan']
 
         if lan not in constants.SUPPORTED_LANGUAGES:
-            raise ValueError(f'{lan} not supported')
+            logging.warning("%s not supported", lan)
```

Comment:
It would be cool if you and @oscardavidrm could figure out a way to include this warning in the response so that it could still be displayed on the frontend without failing the request.

In other words, still do the search as normal and return any results, but if the results are empty and the warning is present, the frontend could display an informational popup about the language.

Reply:
That sounds like something cool, I'll have to think on how to raise the warning without interrupting tho.

Reply:
Yeah, that sounds interesting. If you raise an error, it is pretty straight forward since the frontend will get the same format we agreed upon and it would basically just be displaying it in the popup. 

As per my understanding, this has come with some issues right? If you cannot raise `ValueError`, we could probably look into an additional attribute in your response, a `warnings array` for example containing strings with relevant information (such as the one you're pointing out) and I could look for that additional attribute and group those so called warnings.